Gems/AtomLyIntegration/EMotionFXAtom/Code/Tools/EMStudio/AnimViewportToolBar.cpp

```diff
@@ -144,4 +153,23 @@ namespace EMStudio
             }
         }
     }
+
+    void AnimViewportToolBar::LoadSettings()
+    {
+        AZStd::string renderFlagsFilename(EMStudioManager::GetInstance()->GetAppDataFolder());
+        renderFlagsFilename += "AnimViewportRenderFlags.cfg";
+        QSettings settings(renderFlagsFilename.c_str(), QSettings::IniFormat, this);
+
+        const bool isChecked = settings.value("CameraFollowUp").toBool();
+        m_followCharacterAction->setChecked(isChecked);
```

Comment:
You should pass in a default value as the second parameter to the value function in case the setting does not exist in the file.